entrypoint.sh

```diff
@@ -53,7 +54,9 @@ fi
 
 debug "Starting kubectl collecting output"
 
-if [ -n "${INPUT_STDIN:-}" ]; then
+if [ -n "${INPUT_SCRIPT:-}" ]; then
+  output=$(eval "${INPUT_SCRIPT}")
```

Comment:
instead of `eval`, maybe write it to a file and running that? `eval` is best avoided

(note i have not tested this)
```suggestion
  script="$(mktemp)"
  echo "$INPUT_SCRIPT" > "$script"
  output=$(/bin/sh "$script")
```